templates/apigee-edge-ui-list.html.twig

```diff
@@ -0,0 +1,108 @@
+{#
+/**
+ * @file
+ * Default theme implementation to display an App list.
+ *
+ * Available variables:
+ * - attributes: HTML attributes to apply to the main <div> tag.
+ * - header: The App listing header cells. Each cell contains the following properties:
+ *   - tag: The HTML tag name to use; either 'th' or 'td'.
+ *   - attributes: HTML attributes to apply to the tag.
+ *   - content: A localized string for the title of the column.
+ *   - field: Field name (required for column sorting).
+ *   - sort: Default sort order for this column ("asc" or "desc").
+ * - header_attributes: HTML attributes to apply to the header.
+ * - items: The array of Apps. Each app contains the following properties:
+ *   - attributes: HTML attributes to apply to the App.
+ *   - team: The Team that the App belongs to (only available on /team-apps path)
+ *   - name: The App's name.
+ *   - status: The App's status in a status_property render array.
+ *   - operations: Link which can be accessed in accordance with the App.
+ *   - warning_message: The App's warning message if there's any.
+ *   - warning_attributes: The warning message's attributes.
+ * - items_attributes: HTML attributes to apply to the items.
+ * - show_collapse_link: indicates whether the Hide/Show links should be there or the warning message should always be
+ *   visible under the App info.
+ * - show_text: The test to be shown when the Hide/Show links are there in closed state.
+ * - hide_text: The test to be shown when the Hide/Show links are there in open state.
+ * - empty: The message to display in an extra row if table does not have
+ *   any rows.
+ * - empty_attributes: HTML attributes to apply to the empty list.
+ *
+ * @see template_preprocess_app_list()
+ *
+ * @ingroup themeable
+ */
+#}
+{{ attach_library('apigee_edge_ui/apigee_edge_ui.lists') }}
+{%
+  set classes = [
+    'entity-list',
+    type ? 'entity-list--' ~ type
+  ]
+%}
+<div {{ attributes.addClass(classes) }}>
+
+  {# Items start #}
+  {% if items %}
+
+    {# Header start #}
+    {% if header %}
+      <div{{ header_attributes }}>
+        {% for cell in header %}
+          <div{{ cell.attributes }}>
+            {{- cell.content -}}
+          </div>
+        {% endfor %}
+      </div>
+    {% endif %}
+    {# Header end #}
+
+    <div{{ items_attributes }}>
+      {% for item in items %}
+
```

Comment:
These empty lines are going to appear in the generated HTML output too,aren't they? Are they really necessary?

Reply:
No these won't appear in the markup, I added these for better readability.